### PR TITLE
Allow extlib version 1.x

### PR DESCRIPTION
**Pull request by ekohl**
Closes GH-317

### CODE REVIEW

**Comment by ekohl** [comment]
This mirrors https://github.com/theforeman/puppet-puppet/commit/097d0a8adff438426687beef3990b6b488a1751a

File: .fixtures.yml [hunk 1]
```diff
@@ -4,9 +4,7 @@ fixtures:
     concat:   'git://github.com/puppetlabs/puppetlabs-concat'
     dhcp:     'git://github.com/theforeman/puppet-dhcp'
     dns:      'git://github.com/theforeman/puppet-dns'
-    extlib:
-      repo: 'git://github.com/puppet-community/puppet-extlib'
-      ref:  'v0.11.3'
+    extlib:   'git://github.com/puppet-community/puppet-extlib'
```
Comment (domcleal) [inline .fixtures.yml]: Could you change the org/owner to `voxpupuli` too please?

Comment (ekohl) [inline .fixtures.yml]: Updated

**Comment by mmoll** [comment]
merged, bedankt @ekohl!

**Pull request merged**